.mocharc.json

```
@@ -0,0 +1,5 @@
+{
+    "extension": ["ts"],
+    "spec": "src/**/*.spec.ts", 
```

que tal usar o sufixo `test` ao inves de `spec`? Acho que fica mais claro que são arquivos de teste e vc? 
```suggestion
    "spec": "src/**/*.test.ts", 
```